src/containers/PaletteForm/PaletteForm.js

```diff
@@ -100,11 +100,13 @@ export class PaletteForm extends Component {
             value={this.state.paletteName}
             onChange={this.handleChange}
           />
-          <DropdownButton id="dropdown-custom-1" title="Add To Project">
-            <Dropdown.Item className="dropdown-item" onClick={this.newProject}>Add new project</Dropdown.Item>
-            <div className="dropdown-divider"></div>
-            {this.showProjects()}
-          </DropdownButton>
+          { this.state.paletteName.length >= 1 &&
```

Comment:
Great job find and fixing this bug. I like the use of conditional rendering to disallow users to add prior to having a palette name.